Articles/Adding-metadata-to-tickets-created-by-the-Mobile-SDK.md

```diff
@@ -0,0 +1,124 @@
+* **Ticket forms (Enterprise only):** A ticket form is a set of predefined ticket fields for a specific support request. For more information please see [Creating Ticket Forms](https://support.zendesk.com/hc/en-us/articles/203661616-Creating-ticket-forms-to-support-multiple-request-types-Enterprise-).
+
+
+### Ticket Fields
+One important pre-requisite before adding ticket fields to requests created in the mobile SDKs is to make said ticket fields editable by end-users, as all of the Mobile SDKs interactions with Zendesk are done so from the perspective of an end user. If you do not do this the values set in the requests will be discarded. This can be done in Admin -> Ticket Fields (Under the Manage subheading) and clicking edit beside the relevant ticket field. From here you can make the field editable by end-users.
+
+![Ticket fields admin pages](images/ticket_fields_admin.png)
+
+#### iOS
+Ticket fields are set globally in the iOS SDK so they will apply to all the tickets a user submits using the iOS mobile SDK. To do so you need to set ```ZDKCustomField``` objects in the customTicketFields property of ```ZDKConfig```. 
+
+A ```ZDKCustomField``` object consists of a field ID and a value. The field ID can be found in your Zendesk. Simply edit the custom field and the ID will be shown at the top of the page, as shown below. 
+
+![Ticket fields admin page](images/ticket_fields_admin_2.png)
+
+Once you have the ID it is simply a matter of providing whatever information you wish to show and adding it to ```ZDKConfig```. In the case of drop down fields it is important the value you set is the exact string used in the field options of the ticket field. A complete example is below:
+
+```objective-c
+
+	//Device free space
+	NSString *deviceFreeSpace = [NSString stringWithFormat:@"%f GB", [ZDKDeviceInfo freeDiskspace]];
+	ZDKCustomField *customFieldDeviceFreeSpace = [[ZDKCustomField alloc] initWithFieldId:@(00000000) andValue:deviceFreeSpace];
+ 
+	//Device battery level
+	NSString *deviceBatteryLevel = [NSString stringWithFormat:@"%f", [ZDKDeviceInfo batteryLevel]];
+	ZDKCustomField *customFieldDeviceBatteryLevel = [[ZDKCustomField alloc] initWithFieldId:@(00000001) andValue:deviceBatteryLevel];
+
+	[ZDKConfig instance].customTicketFields = @[customFieldDeviceFreeSpace, customFieldDeviceBatteryLevel];
+	
+```
+#### Android
+```java
+
+	String appVersion = String.format(Locale.US, "Version %s", BuildConfig.VERSION_NAME);
+	CustomField customFieldAppVersion = new CustomField(0l, appVersion);
+
+	String osVersion = String.format(Locale.US, "Android %s, Version %s", VERSION.RELEASE, VERSION.SDK_INT);
+	CustomField customFieldOsVersion = new CustomField(1l, osVersion);
+
+	ZendeskConfig.INSTANCE.setCustomFields(Arrays.asList(customFieldAppVersion, customFieldOsVersion));
+	
+```
+
+### Tags
+Tags are a very simple way to give context to and organize tickets. 
```

Comment:
I think it would be better to explain this better, an intro to what the code is doing, when it will take effect, is it for all tickets that are request, is it just for that one instance?